custom/icds_reports/mpr_sqldata.py

```diff
@@ -1568,7 +1568,7 @@ def row_config(self):
                     'alias': 'M_sev_mod_resident_weighted'
                 },
                 {
-                    'columns': ('F_sev_mod_resident_weighted', 'M_sev_mod_resident_weighted'),
+                    'columns': ('F_sum_sev_resident_weighted', 'M_sev_mod_resident_weighted'),
```

Comment:
THis was summing up `F_sev_mod_resident_weighted` and `M_sev_mod_resident_weighted`. 
But Since  there is not alias created as `F_sev_mod_resident_weighted` so the sum comming out was just value of `M_sev_mod_resident_weighted`.  The sum needs to be sum of `F_sum_sev_resident_weighted` and `M_sev_mod_resident_weighted` which are calculated aboved.

@snopoke  fyi